src/app/page.tsx

```diff
@@ -11,13 +11,19 @@ export default async function Home() {
   const globalRepo = container.resolve<GlobalDataRepository>('GlobalDataRepository');
   const countryMapDataPromise = globalRepo.getMapDataForCountries();
   const disputedAreasPromise = globalRepo.getDisputedAreas();
-  const [countryMapData, disputedAreas] = await Promise.all([countryMapDataPromise, disputedAreasPromise]);
+  const ipcDataPromise = globalRepo.getIpcData();
+  const [countryMapData, disputedAreas, ipcData] = await Promise.all([
+    countryMapDataPromise,
+    disputedAreasPromise,
+    ipcDataPromise,
+  ]);
+
```

Comment:
@Tschonti we should not fetch the IPC Data here right? Any suggestions where to move this, so that we fetch the data only on sidebar click on the ipc icon

Reply:
Yeah it'd be great if we didn't fetch it here, but it was done so the mapbox layers and interactivity can be set up with this data. I'm not sure if mapbox is okay with dynamically creating a new layer (though it probably is).

the IPC data endpoint is ~21 KB, which is not that small, but way smaller than adm0 for example, so I don't think it makes a big difference to prefetch this.